src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -218,7 +218,7 @@ private async Task InitializeAsync()
             _solutionEvents.ProjectRemoved += OnEnvDTEProjectRemoved;
             _solutionEvents.ProjectRenamed += OnEnvDTEProjectRenamed;
 
-            var vSStd97CmdIDGUID = VSConstants.GUID_VSStandardCommandSet97.ToString("B", CultureInfo.CurrentCulture);
+            var vSStd97CmdIDGUID = VSConstants.GUID_VSStandardCommandSet97.ToString("B", null);
```

Comment:
Optional: Consider adding named param for null arguments. This applies to al GUID arguments.
```suggestion
            var vSStd97CmdIDGUID = VSConstants.GUID_VSStandardCommandSet97.ToString("B", provider: null);
```